notification/views.py

```diff
@@ -21,6 +21,8 @@ def get_resource_objects(self):
     def get(self, request, *args, **kwargs):
         self.notification.seen = True
         self.notification.save()
+        submission = self.notification.submission
+        submission.exercise.grade(submission)
```

Comment:
What does this line here do? It's purpose seems somewhat unclear to me.

Reply:
This sends a POST request to Jutut that makes Jutut update the feedback submission information, including the feedback_response_seen field. Jutut gets this information from A+ API.

Reply:
Ah, that makes sense! Perhaps it could also be commented in the code shortly, so if in the future someone else wonders about it, it's easier to understand the purpose of this line.